src/main/java/com/synopsys/integration/jenkins/scan/extension/pipeline/SecurityScanStep.java

```diff
@@ -75,6 +76,8 @@ public class SecurityScanStep extends Step implements Serializable {
     private String synopsys_bridge_path;
     private Boolean bridge_include_diagnostics;
 
+    private Boolean BRIDGE_NETWORK_AIRGAP;
```

Comment:
Need to check whether we should make this lowercase like other parameters.

Reply:
Done